src/Service.Tests/Unittests/RuntimeConfigLoaderJsonDeserializerTests.cs

```diff
@@ -121,7 +121,8 @@ public void CheckCommentParsingInConfigFile()
                                             ""set-session-context"": true
                                         },
                                     ""connection-string"": ""Server=tcp:127.0.0.1,1433;Persist Security Info=False;Trusted_Connection=True;TrustServerCertificate=True;MultipleActiveResultSets=False;Connection Timeout=5;""
-                                    }
+                                    },
```

Comment:
We changed entities to be a non-nullable type on the constructor and hence this is required. Otherwise it takes entities as null and then when it does the iteration it will throw null reference exception.